offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/offergeneration/QuotationDetails.groovy

```diff
@@ -284,9 +357,9 @@ class QuotationDetails {
      * It also defines the acronyms used to abbreviate the product groups in the offer listings.
      */
     enum ProductGroup {
-        DATA_GENERATION("Data generation", "DG"),
-        DATA_ANALYSIS("Data analysis", "DA"),
-        PROJECT_AND_DATA_MANAGEMENT("Project management & data storage", "PM & DS")
+        DATA_GENERATION("Data Generation", "DG"),
+        DATA_ANALYSIS("Data Analysis", "DA"),
+        PROJECT_AND_DATA_MANAGEMENT("Project Management & Data Storage", "PM & DS")
```

Comment:
Actually this is not related to the fix 🤓

Reply:
I have overlooked that ;)

Reply:
That's completely fine, but since we generate the title dynamically it's necessary to adjust this in this fix 🔧